tests/TestCase/Command/CacheCommandsTest.php

```diff
@@ -42,7 +42,7 @@ public function setUp()
      *
      * @return void
      */
-    public function tearDown()
+    public function tearDown(): void
```

Comment:
Do we have sniffer rules that do this work or should we plan to add this to rector when we get to that point?

Reply:
Strict one of slevomatik could add those, but if enabled it also might add ones we dont want.
Better to run this manually from time to time maybe.

Reply:
I was more thinking about the upgrade path for applications and plugins, as I think all those builds/tests will now fail.

Reply:
There are [rectors](https://github.com/rectorphp/rector/blob/master/docs/AllRectorsOverview.md#phpunit) available for updating the various assert methods.